test/functional/uri_tests.js

```diff
@@ -180,7 +180,7 @@ describe('URI', function() {
         if (err) console.dir(err);
         expect(err).to.not.exist;
         expect(client).to.exist;
-        expect(client.s.options.replicaSet).to.exist.and.equal(config.replicasetName);
+        expect(client.s.options.replicaSet).to.exist.and.equal(String(config.replicasetName));
```

Comment:
What is happening here?

Reply:
There were some instances where `client.s.options.replicaSet` would be `undefined`, but as a string. This change helped those cases pass.

Reply:
That seems like an error so we should undo this change and figure out what is causing `client.s.options.replicaSet` to be `undefined`. Since `client.s.options` comes from `mongo_client.js`, it seems like maybe the test runner isn't passing on the correct `replicasetName` here or is perhaps not filtering correctly.

Reply:
undid the change